src/scripts/ui/GameplayOptionPage.tsx

```diff
@@ -109,6 +109,29 @@ export function GameplayOptionPage(): React.ReactNode {
                />
                <div className="sep10" />
             </fieldset>
+            <fieldset>
```

Comment:
I am not sure here's the best place to put the settings. As autopilot needs to be unlocked by tech.

Maybe put it in the WarehouseBuildingBody where it already checks for autopilot? You might want to add a warning that says "This setting applies to all warehouses"

Reply:
Good point, I hadn't though about that.

Reply:
Option moved to the warehouse.